### ISSUE TITLE
0.5.1 and 0.5.2 do not work with kotlin 1.1.51

### ISSUE THREAD

**Issue by robstoll** [raniejade/spek-idea-plugin#56]
I updated the spek-idea-plugin to 0.5.2. I cannot run a single Spek, I always get the following error:
```
WARNING: TestEngine with ID 'spek' failed to discover tests
java.lang.IllegalAccessError: tried to access method org.junit.platform.engine.support.descriptor.ClassSource.<init>(Ljava/lang/Class;)V from class org.jetbrains.spek.engine.SpekTestEngine
```
The same happens with 0.5.1

**Comment by raniejade** [comment]
What version of Spek and JUnit Platform are you using? I updated the bundled JUnit Platform used by the plugin. If you can also paste your build script, please do so. Thanks!

**Comment by raniejade** [comment]
Looks like some incompatible versions of JUnit Platform are in the classpath.

**Comment by robstoll** [comment]
spek version 1.1.4 and junit platform 1.0.0-RC3,
I update to plugin 0.5.3, still the same problem.
I update junit platform to 1.0.1 still the same problem.
I update spek to 1.1.5 => problem solved

I downgraded back to junit-platform 1.0.0-RC3 again, still with spek 1.1.5 => problem solved. 
So it seems that the plugin is not compatible with spek 1.1.4

The problem is solved for me, since I can update to 1.1.5. Maybe you want to look further into it. Otherwise I am fine if you close this issue

**Comment by raniejade** [comment]
I'm closing this for now, there has been a lot of compatibility issues with junit platform :(
